apps/ledger-live-mobile/src/icons/GooglePay.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React from "react";
+import Svg, { Path } from "react-native-svg";
+
+type Props = {
+  height: number,
+  width: number,
+};
+
+export default function GooglePay({ height = 12, width = 32 }: Props) {
+  return (
+    <Svg viewBox="0 0 32 12" width={width} height={height}>
```

Comment:
Svg can be complex for overall performance. We should aim to do one of these:
- keep using svg but memo that component
- use an asset instead. static image instead of component.

Reply:
Btw, is there a reason we don't use https://react-svgr.com/ ?
With it we can generate components with memo from svg files and it supports RN